src/storages/ConfigStorage.sol

```diff
@@ -653,6 +656,42 @@ contract ConfigStorage is IConfigStorage, OwnableUpgradeable {
     }
   }
 
+  function addStepMinProfitDuration(StepMinProfitDuration[] memory _stepMinProfitDurations) external onlyOwner {
+    uint256 length = _stepMinProfitDurations.length;
```

Comment:
If we're not gonna have remove func, consider having max step cap.

Reply:
Revised these function